k8s/services/rabbitmq/helm-3.4.cue

```diff
@@ -0,0 +1,294 @@
+package rabbitmq
+
+import (
+	"k8s.io/api/core/v1"
+	"guku.io/devx/k8s"
+
+)
+
+#KubeVersion: [=~"^3\\.4\\."]: minor: >=21
+#Values: [=~"^3\\.4\\."]: {
+	global: {
+		imageRegistry: ""
+		imagePullSecrets: [...v1.#LocalObjectReference]
+		storageClass: ""
+	}
+
+	kubeVersion:       #KubeVersion
```

Comment:
are you sure this is right? we invented the `#KuberVersion` schema for our usage